doc/content/source/userobjects/NumberDensityAccumulator.md

```diff
@@ -0,0 +1,36 @@
+# NumberDensityAccumulator
+
+!syntax description /UserObjects/NumberDensityAccumulator
+
+## Overview
+
+In FENIX the charge density $\rho$ is defined as
+
+\begin{equation} \label{eq:charge_density}
+  \rho
+  \left(
+    \vec{r}\,
+  \right)
+   =
+  \sum_{i=1}^N
+  q_i
+  w_i \delta
+  \left(
+    \vec{r} - \vec{r}_i
+  \right),
+\end{equation}
+
+where $N$ is the number of computational particles in the simulation, $q_i$ is the charge of the $i^\text{th}$ particle, $w_i$ is the weight of the $i^\text{th}$ particle, and $\vec{r}_i$ is the position at which the $i^\text{th}$ computational particle exists.
+
+## Visualization
+
+
```

Comment:
```suggestion
```

Did you mean to add a figure? This section is empty. Suggesting a removal in case you didn't mean to leave it and want to batch it up with the rest of the comments.